bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieNettyServer.java

```diff
@@ -279,6 +280,12 @@ void start() {
     void shutdown() {
         LOG.info("Shutting down BookieNettyServer");
         isRunning.set(false);
+
+        if (!isClosed.compareAndSet(false, true)) {
```

Comment:
Same thing applied to isRunning. If the shutdown will be called twice, I would like it to be set eventually.